addons/mail/models/mail_activity.py

```diff
@@ -701,6 +701,8 @@ def _default_activity_type(self):
         help="Type of the exception activity on record.")
     activity_exception_icon = fields.Char('Icon', help="Icon to indicate an exception activity.",
         compute='_compute_activity_exception_type')
+    my_activity_date_deadline = fields.Date(compute='_compute_my_activity_date_deadline',
```

Comment:
Linting: would move this definition under 'activity_date_deadline' to ease model reading.